app/schemas/com.jerboa.db.AppDB/25.json

```diff
@@ -2,7 +2,7 @@
   "formatVersion": 1,
   "database": {
     "version": 25,
-    "identityHash": "e7e957b63c2297483c5e9aeb1e263a1f",
```

Comment:
Increase the DB version, do not change the existing scheme

Reply:
Also add migrations and test them

Reply:
You should see this reflected in my latest commit. The migration test passed.